src/libstd/process.rs

```diff
@@ -701,7 +701,14 @@ impl AsInnerMut<imp::Command> for Command {
     fn as_inner_mut(&mut self) -> &mut imp::Command { &mut self.inner }
 }
 
-/// The output of a finished process.
+/// The output of a finished process. This is returned in a Result by
```

Comment:
Could you put some newlines between the first and second sentences here? The first sentence operates as a summary, so it's helpful to add space.

```rust
/// The output of a finished process.
///
/// This is returned in a Result...
```

Reply:
Done. I think that does look neater :)